src/storage/v2/storage.cpp

```diff
@@ -88,6 +90,34 @@ Storage::Accessor::Accessor(Accessor &&other) noexcept
 
 StorageMode Storage::GetStorageMode() const noexcept { return storage_mode_; }
 
+std::vector<EventInfo> Storage::GetMetrics() noexcept {
+  std::vector<EventInfo> result;
+  auto info = this->GetBaseInfo();
+
+  for (auto i = 0; i < memgraph::metrics::CounterEnd(); i++) {
+    result.emplace_back(memgraph::metrics::GetCounterName(i), memgraph::metrics::GetCounterType(i), "Counter",
+                        memgraph::metrics::global_counters[i].load(std::memory_order_acquire));
```

Comment:
no need to use memgraph::metrics, use just metrics::
and I think you don't need to use load, without load should suffice